tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = test
+envlist = test{py27,py33,py34}
```

Comment:
ah, i missed the lack of hyphen

if we go with the first version we need to drop the `test:` prefix from the `deps` section

also need to update `.travis.yml` accordingly